jdaviz/configs/imviz/tests/test_linking.py

```diff
@@ -281,26 +283,34 @@ def test_pixel_linking(self):
         label_mouseover = self.imviz.app.session.application._tools['g-coords-info']
         label_mouseover._viewer_mouse_event(self.viewer,
                                             {'event': 'mousemove', 'domain': {'x': -1, 'y': -1}})
-        assert label_mouseover.as_text() == ('Pixel x=-1.0 y=-1.0',
-                                             'World 00h14m19.5987s -30d23m31.0683s (ICRS)',
-                                             '3.5816611274 -30.3919634282 (deg)')
+
+        # do not display world coordinates outside of the bounding box
+        assert label_mouseover.as_text() == ('Pixel x=-1.0 y=-1.0', '', '')
         assert not label_mouseover.row1_unreliable
-        assert label_mouseover.row2_unreliable
         assert label_mouseover.row3_unreliable
 
+        # `row2_unreliable` was True when the WCS-only layer used GWCS.
+        # now with FITS WCS as the coordinate frame, there is no bounding
+        # box and row2 is reliable.
+        assert not label_mouseover.row2_unreliable

```

Comment:
What does "unreliable" even mean anymore now that we do not allow any extrapolation? Should this whole "unreliable" definition be removed? Maybe @kecnry can chime in too since I think he coded a lot of these.

Reply:
If we _never_ set it to True anymore, then we can remove it.  If there are cases that use it, we might as well keep it.

Reply:
Where do we use it if we now follow GWCS bounding box and anything unreliable is always NaNs?